apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/common/ClaimDao.java

```diff
@@ -163,7 +235,6 @@ Predicate createMbiPredicate(
       Path<?> root,
       String mbiSearchValue,
       boolean isMbiSearchValueHashed,
-      boolean isOldMbiHashEnabled,
```

Comment:
Need to update javadoc

Reply:
Thanks.  I have fixed that and a few more.